update.py

```diff
@@ -548,12 +548,12 @@ def _updateNumbers(python):
     cases_dir = getTestCasesDir()
 
     for filename in sorted(os.listdir(cases_dir)):
-        if filename == "InplaceOperationInstanceStringAdd.py":
+        if filename in ["InplaceOperationInstanceStringAdd.py", "RepeatedFunctionCall.py.j2"]:
```

Comment:
This change seems unnecessary now?

Reply:
It`s unnecessary until [THIS](https://github.com/Nuitka/Nuitka/pull/350) pull request is not merged, but then we will get an error while measuring construct performance with this filename